plugins/resource-backup.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: resource-backup
+spec:
+  version: v0.1.0
+  homepage: https://github.com/zak905/kubectl-resource-backup
+  shortDescription: back up K8 resources (like CRDs) to local disk
```

Comment:
> Save Kubernetes resources to disk

is better. 

K8 is not a thing.

Reply:
Sure

Reply:
Did you fix this part? This is not showing up in the diff.

Reply:
oops sorry, I still don't know what happened. the commit was in my fork's branch but still it did not show up here in the PR. Fixed by rebasing against master and force pushing.